libdl/VITABUILD

```diff
@@ -0,0 +1,23 @@
+pkgname=libdl
+pkgver=r1.bf46551
+pkgrel=1
+url="https://github.com/hyln9/vita-libdl"
+source=("git+https://github.com/hyln9/vita-libdl.git#commit=bf46551882c344e5aebee07f15267e2ca93d78a5")
+sha256sums=('SKIP')
+depends=('taihen')
+
+pkgver() {
+  cd vita-libdl
+  printf "r%s.%s" "$(git rev-list --count HEAD)" "$(git rev-parse --short HEAD)"
```

Comment:
Can you remove this?

Reply:
How would you like to see this versioned? I don't see a version number in the repository itself, so I ended up taking the repo commit number.

Reply:
I ended up setting the version to 1. Let me know if you'd like to see it in a different way.